src/ReactiveUI.Uno/ReactiveUI.Uno.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>xamarinmac20;MonoAndroid90;xamarinios10;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.17763</TargetFrameworks>
+    <PackageId>ReactiveUI.Uno</PackageId>
+    <Title></Title>
+    <Description>Uno-Platform specific extensions to ReactiveUI</Description>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>  
+  <PropertyGroup>
+    <DefineConstants>HAS_UNO</DefineConstants>
```

Comment:
This is automatically defined by the Uno package. See here:

https://github.com/nventive/Uno/blob/d9d01e7a06f289d1188807573d3ce5487db0a35d/build/uno.ui.targets#L23

Reply:
Left it in Debug